assets/multiqc_config.yml

```diff
@@ -98,22 +99,26 @@ top_modules:
         - "*raw*"
       path_filters_exclude:
         - "*processed*"
-      extra: "If used in this run, Falco is a drop-in replacement for FastQC producing the same output, written by Guilherme de Sena Brandine and Andrew D. Smith."
+      extra: "If used in this run, Falco is a drop-in replacement for FastQC producing
```

Comment:
@jfy133 Not sure why there are so many spaces and new lines. It might be due to some weird behavior in my VS Code?

Reply:
Something has indeed gone weird, this doesn't look right...